example-dvc-experiments/.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.dvc linguist-language=YAML
```

Comment:
This was renamed to `example-dvc-experiments`, so it's missing the `.gitattributes` copy that I had in https://github.com/iterative/example-repos-dev/commit/a28bbb5bb91694cd9a713038a491f712b3bc4b19.

Reply:
I'll make a separate PR for that.